lib/Zonemaster/Backend/DB/SQLite.pm

```diff
@@ -79,27 +79,27 @@ sub create_db {
     ) or die "SQLite Fatal error: " . $self->dbh->errstr() . "\n";
```

Comment:
Here is a die, but not for the other tables or indexes. Why not everywhere?

Reply:
Indeed this is something I forgot in the process. And this die has been added long time ago (https://github.com/zonemaster/zonemaster-backend/commit/cad366f845a71ae30efa9257b68b5b0766b4cb61).